app/javascript/app/routes.js

```diff
@@ -323,6 +325,29 @@ export default [
           }
         ]
       },
+      {
+        path: '/emission-pathways/scenarios/:id',
+        component: EmissionPathwaysScenario,
+        label: 'EMISSION PATHWAYS SCENARIO',
+        headerImage: 'emission-pathways',
+        sections: [
+          {
+            hash: 'indicators',
+            label: 'Indicators',
+            anchor: true,
+            nav: true,
+            component: EmissionPathwaysTableMenu
```

Comment:
I am not sure you need this section. You are wanting to render the table component, this table has no menu also. The anchor link sections only need to know which component is rendering. If you had a menu you can just place is inside the table component and then render the table. In this particular case all we need is a title "Indicators" not a menu so it seems over kill to me.